examples/solids/README.rst

```diff
@@ -0,0 +1,84 @@
+
+To set the ExodusII mesh file, use the :code:`-mesh` option and the file path to the mesh file.
+
+To set the polynomial degree of the finite element basis, use the :code:`-degree` option.
+
+To set the materiel parameters, use the :code:`-E` and :code:`-nu` options, for Young's modulus and Poisson's ratio, respectively.
+
+To set the boundary conditions, use :code:`-bc_zero` or :code:`-bc_clamp` followed by a comma separated list of the constrained faces.
+
+A face set with :code:`-bc_zero` will remain fixed at zero displacement and a face set with :code:`-bc_clamp` will be displaced by :code:`-bc_clamp_max` in the y direction, or -1 in the y direction if this value is not set.
+
+The following is an example of a minimal set of command line options::
+
+   ./elasticity -mesh ./meshes/meshes/cylinder8_672e_4ss_us.exo -degree 4 -E 1e6 -nu 0.3 -bc_zero 999 -bc_clamp 998
+
+These command line options are the minimum requirements for the mini-app, but additional options may also be set.
+For example, the problem formulation (:code:`-problem`), forcing term (:code:`-forcing`), libCEED backend resource (:code:`-ceed`) can be specified::
+
+   ./elasticity -mesh ./meshes/meshes/cylinder8_672e_4ss_us.exo -degree 4 -E 1e6 -nu 0.3 -bc_zero 999 -bc_clamp 998 -problem hyperFS -forcing none -ceed /cpu/self/opt/blocked
+
+Available runtime options are:
+
++-----------------------+-------------------------------------------------------------------+
+| Option                | Meaning                                                           |
++=======================+===================================================================+
+| ``-ceed``             | CEED resource specifier                                           |
++-----------------------+-------------------------------------------------------------------+
+| ``-ceed_fine``        | CEED resource specifier for fine grid (P >= 5)                    |
++-----------------------+-------------------------------------------------------------------+
+| ``-test``             | Run in test mode                                                  |
++-----------------------+-------------------------------------------------------------------+
+| ``-degree``           | Polynomial degree of tensor product basis                         |
++-----------------------+-------------------------------------------------------------------+
+| ``-mesh``             | Filepath to Exodus-II file                                        |
++-----------------------+-------------------------------------------------------------------+
+| ``-problem``          | Problem to solve (``linElas``, ``hyperSS``, or ``hyperFS``)       |
++-----------------------+-------------------------------------------------------------------+
+| ``-forcing``          | Forcing term option (``none``, ``constant``, or ``mms``)          |
```

Comment:
How do we feel about explaining what `mms` stands for? I would have personally chosen another name for it, since the method of manufactured solutions is a method, not a type of forcing. But if you really like that name, at least, please explain the acronyms you choose for the people who did not follow the entire development and are going to read this later on some day.

Reply:
I added an explanation. "the forcing term to use when running the method of manufactured solutions to verify the convergence of the linear formulation" was a bit lengthy to make as an option which is why we used mms : )

Reply:
Hum... the other two options denote a type of forcing (`none` or `constant`, not a method). I would have tried to be consistent and just used a word that describes the type of forcing for this case, perhaps `nonlinear`? All other examples we have for our BP3s adopt the method of manufactured solutions, but we never say that those examples serve as a code-validation tool, to verify that our numerical approximation can adequately reproduce the continuous operator. Instead we say, we solve Poisson's problem. Do you see what I mean?

Reply:
`mms` also denotes a forcing term. This option sets the forcing term to be the one analytically computed from the manufactured solution. `nonlinear` provides less information about the forcing term.

Reply:
s/validation/verification/

I don't have a problem with `-forcing mms` as terminology. The fluids solver primarily solves the density current configuration (we need to make it more flexible). It would be really useful for the fluids solver to have good manufactured solutions, such as [MASA](https://github.com/manufactured-solutions/MASA), in which case something like `-forcing mms` or `-problem mms` would make sense. The other examples are currently toys so they _only_ do manufactured/analytic solutions.